BEIMA.Client/src/pages/DeviceTypes/DeviceTypesPage.js

```diff
@@ -13,34 +14,19 @@ const DeviceTypesPage = () => {
     setPageName('Device Types')
   },[setPageName])
  
-  const mockCall = async () => {
-    const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
-    await sleep(1000)
-    let data = []
-    for(let i = 0; i < 5; i++){
-      data.push({
-        deviceTypeId: i,
-        name: `Test Item Type #${i}`,
-        description: "A short informational description",
-        numDevices: i+4,
-      })
-    }
+  const DevicesCall = async () => {
+    let deviceTypeData = await GetDeviceTypeList();
+    
     // Map data into format supported by list
-    let mapped = data.map(({deviceTypeId, name, description, numDevices}) => {
-      return {
-        id: deviceTypeId,
-        name: name,
-        description: description,
-        numDevices: numDevices
-      }
-    })
-    return mapped
+    let data = deviceTypeData.response.map((item) => { return { id: item.id, name: item.name, description: item.description, notes: item.notes, lastModified: item.lastModified, buildingName: "building name needed", numDevices: "count at DB or remove? counting length on the DT return gives count of device type"} });
```

Comment:
I think we can do without this filtering of the response since the GetDeviceTypeList already filters the data returned from the endpoint. We should be able to call the GetDeviceTypeList straight from the useEffect and get everything we need. It will need updated once we get the endpoint to return the count, but we can worry about that later.

Reply:
I had it set that way to have something for the other fields, based on comments for the tag from Joseph and this one I've changed it to just return the response so any updates will be passed through